segtypes/common/codesubsegment.py

```diff
@@ -54,7 +54,14 @@ def scan_code(self, rom_bytes, is_asm=False):
             self.get_exclusive_ram_id(),
         )
 
-        self.spim_section.isHandwritten = is_asm
+        for symbol_list in self.seg_symbols.values():
```

Comment:
Reminder to myself: remove those extra for loops, I added them back because I got a bit confused while merging master.